ipn/ipnlocal/filesharingstatus_windows.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//+build windows
+
+package ipnlocal
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"golang.org/x/sys/windows/registry"
+)
+
+const (
+	sendFileShellKey = "*\\shell\\tailscale"
+)
+
+var (
+	ipnExePath = ""
+)
+
+// Should be called from a locked code
```

Comment:
I would just put a mutex around ipnExePath and not worry about documenting the requirements?

Reply:
I've thought about that, but the function is currently called only from locked code anyway, so concluded that there's no reason to lock within lock, kinda overkill. But OK, I'll add.